nix/default.nix

```diff
@@ -24,6 +24,8 @@ with ocamlPackages; buildDunePackage rec {
     mirage-crypto-ec
     mirage-crypto-rng
     secp256k1-internal
+    cohttp
+    cohttp-lwt-unix
```

Comment:
I had a Piaf version initially but it wasn't working (POST requests were failing silently). I'll reproduce it so we can decide.